lib/fastly/acl.rb

```diff
@@ -0,0 +1,25 @@
+class Fastly
+  class ACL < BelongsToServiceAndVersion
+    attr_accessor :service_id, :name, :old_name
+
+    ##
+    # :attr: service_id
+    #
+    # The id of the service this belongs to.
+
+    ##
+    # :attr: version
+    #
+    # The number of the version this belongs to.
+
+    ##
+    # :attr: name
+    #
+    # The name of the response object
+
+    ##
+    # :attr: old_name
+    #
+    # The name of the response object to update
+  end
```

Comment:
How do we add an `ACLEntry` to an `ACL`? I believe some CRUD methods might be required in here. You might take a look at `Dictionary`/`DictionaryItem` for some ideas:
https://github.com/fastly/fastly-ruby/blob/master/lib/fastly/dictionary.rb
https://github.com/fastly/fastly-ruby/blob/master/lib/fastly/dictionary_item.rb